include/rosconsole_bridge/bridge.h

```diff
@@ -46,6 +46,7 @@ class OutputHandlerROS : public console_bridge::OutputHandler
 {
 public:
   OutputHandlerROS(void);
+  ~OutputHandlerROS();
```

Comment:
Thanks @rhaschke for running the checker. I thought the same but found some "answers" which made me worry and wanted another set of eyes:
* http://stackoverflow.com/questions/5746076/in-c-does-overriding-an-existing-virtual-function-break-abi
* http://stackoverflow.com/questions/7351476/virtual-override-and-binary-compatibility

This can go into Indigo and Jade then - no branching :relieved: